### ISSUE TITLE
"Code" button producing warning in mapApp()

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#567]
When I click the "Code" button in `mapApp()` I receive the following error. Note that this error occurs before merging the major changes from the polygon branch into the develop branch.

```R
Warning in "core" %in% state$view && "bgc" %in% state$view && "deep" !=  :
  'length(x) = 3 > 1' in coercion to 'logical(1)'
```

**Comment by dankelley** [comment]
This likely has to do with `&` versus `&&`.  The former accepts (equal-length) vectors on each side.  The latter demands unit-length values on each side.  This used to be ignored by R, then it became a warning, and then it became an error.  Oce was in danger of being booted from CRAN because of this error -- I had to do a quick release.

**Comment by j-harbin** [comment]
Fixed in commit 245cc4358031ab1d7285f28e071f34051ca461cf of the polygon branch. In summary I changed `"deep" != state$view `to be  `!("deep" %in% state$view)` (and the same throughout)
